dm/templates/instance_template/instance_template.py

```diff
@@ -137,6 +142,20 @@ def generate_config(context):
         'description'
     )
 
+    set_optional_property(
+        instance_template['properties'],
+        properties,
+        'sourceInstance',
+        'sourceInstance'
+    )
+
+    set_optional_property(
+        instance_template['properties'],
+        properties,
+        'sourceInstanceParams',
+        'sourceInstanceParams'
```

Comment:
Why do we have the property name and the rename_to both set when they are the same?